src/troute-config/troute/config/config.py

```diff
@@ -233,4 +233,26 @@ def check_lite_restart_directory(cls, values):
                 assert lite_restart_directory, "lite_restart is present in output parameters, but no lite_restart_output_directory is provided."
         
         return values
-    
\ No newline at end of file
+
+    @root_validator(skip_on_failure=True)
+    def check_nts_dt_stream_output_internal_frequency(cls, values):
+        compute_params = values.get('compute_parameters')
+        output_params = values.get('output_parameters')
+        
+        if compute_params and output_params:
+            # Directly access ForcingParameters
+            forcing_params = compute_params.forcing_parameters
+            stream_output = output_params.stream_output
+
+            if forcing_params and stream_output:
+                nts = forcing_params.nts
+                dt = forcing_params.dt
+                internal_freq = stream_output.stream_output_internal_frequency
+
+                # Perform the check
+                if nts and dt and internal_freq:
+                    result = nts * dt / (internal_freq * 60)
+                    if not result.is_integer():
+                        raise ValueError("UPDATE nts. nts*dt/(stream_output_internal_frequency*60) must be an integer.")
```

Comment:
Can you make the error message a little clearer here? Using plain english rather than variable/equations would be nice.